eng/Directory.Build.Data.targets

```diff
@@ -19,7 +19,7 @@
       <CurrentCsprojPath>$(MSBuildProjectDirectory)</CurrentCsprojPath>
```

Comment:
Is there a reason you needed to put this into a target, instead of just in the props file?

Reply:
It needs to be a dynamic property that generates diruing the build. i.e. just before the particular package is built. Setting it in the prop file will not allow pointing to different readme's.

Reply:
I'm not sure that is true as I think you are using the project property based on which project is being built so I think you can do this in the props file but it isn't a huge deal.